package.json

```diff
@@ -6,6 +6,7 @@
     "lib"
   ],
   "homepage": ".",
+  "proxy": "http://localhost:4142/",
```

Comment:
I'm confused.... isn't this already merged in main? (https://github.com/quantumblacklabs/kedro-viz/pull/318)

Reply:
Yeah it is now, I just needed it here before it was merged a while back. Should just merge on top no problem I think.